src/core/public/i18n/i18n_eui_mapping.test.ts

```diff
@@ -17,6 +17,7 @@ import { getEuiContextMapping } from './i18n_eui_mapping';
 const VALUES_REGEXP = /\{\w+\}/;
 
 describe('@elastic/eui i18n tokens', () => {
+  const i18nTranslateActual = jest.requireActual('@kbn/i18n').i18n.translate;
```

Comment:
I wanted this to actually compare final string outputs to each other as I didn't think it was as useful / readable to try and compare the EUI output to some contrived pre-pluralization Kibana version.